sdks/go/examples/stringsplit/offsetrange/offsetrange.go

```diff
@@ -0,0 +1,107 @@
+package offsetrange
```

Comment:
Please add the Apache 2.0 License header to the new files.

Reply:
Whoops, fixed.